schema/build-config-schema.js

```diff
@@ -436,11 +451,16 @@ rules.forEach(function forRule(rule) {
   }
   if (custom) {
     // @ts-ignore
-    scheme.type = [ "boolean", "object" ];
+    scheme.type = ["boolean", "object"];
     scheme.additionalProperties = false;
   }
-  rule.names.forEach(function forName(name) {
-    schema.properties[name] = scheme;
+  rule.names.forEach(function forName(name, index) {
+    if (index == 0)
+      schema.properties[name] = scheme;
+    else
+    schema.properties[name] = {
+      "$ref": `#/properties/${rule.names[0]}`
```

Comment:
I love this change IFF VS Code handles it properly - do you happen to know? Otherwise, I'd consider this a regression.

Reply:
Yes, it's is a expected VS Code feature. :)